src/content/docs/merchants/storefront-builder/index.mdx

```diff
@@ -1,42 +1,85 @@
+
+Learn how to access and use Product Visuals in the [Product Visuals](/merchants/storefront-builder/product-visuals/) topic.
+
+For more information about AEM Assets, see [Publishing pages with AEM Assets](https://www.aem.live/docs/universal-editor-assets) and [AEM Assets overview](https://experienceleague.adobe.com/en/docs/experience-manager-assets/content/home.html).\
 
 ### Content and Commerce blocks
 
 Blocks are a foundational concept for adding form and function to page sections. They are defined using tables in the content and decorated by your custom code during page rendering. Drop-ins are dynamic, commerce-specific front-end components that can be integrated within a Block.
 
-### Visual Editor (Universal Editor features)
+Learn how to use the Content and Commerce blocks in the [Content and Commerce blocks](/merchants/storefront-builder/content-commerce-blocks/) topic.
 
-Visual Editor is a visual, in-context editing tool that empowers business users and content authors to update storefront content directly on the page, with real-time preview. With Visual Editor, you can make changes to text, images, blocks, and drop-ins without needing to write code or wait for developer assistance. This tool streamlines content workflows, reduces time-to-market for updates, and ensures that your storefront always reflects the latest campaigns, promotions, and brand messaging. Visual Editor is fully integrated with DA.live and supports localization and multistore editing.
+For more information about content blocks, see the [Block Collection](https://www.aem.live/developer/block-collection) topic.
 
-### Product Visuals
+### Edge Delivery Services
 
-[Product Visuals](https://experienceleague.adobe.com/en/docs/commerce/product-visuals/overview) is an integration of Adobe Experience Manager Assets with your Adobe Commerce backend. For Storefront Builder, the digital asset management (DAM) solution accesses editorial assets like images, videos, and other media used on content pages and product enrichments. The DAM integrates with DA.live and Visual Editor for seamless media content management and works with Edge Delivery Services to deliver optimized assets globally.
+Edge Delivery Services is a fast, scalable platform for delivering your storefront's content and assets to customers. It leverages a global content delivery network (CDN) to ensure that pages, images, and other resources load quickly, no matter where your customers are located. By serving content from the edge, EDS reduces latency and improves site performance, which is critical for both user experience and SEO. Edge Delivery Services is tightly integrated with the Storefront Builder ecosystem, ensuring seamless deployment and updates of your storefront content.
```

Comment:
Please double-check if we are still using "EDS." I've been told we are moving away from using the acronyms and advised to use "Edge Delivery Services" instead.